renderer/index.js

```diff
@@ -606,22 +606,22 @@ function addSubtitles (files, autoSelect) {
   if (state.playing.type !== 'video') return
 
   // Read the files concurrently, then add all resulting subtitle tracks
-  console.log(files)
-  var subs = state.playing.subtitles
-  Async.map(files, loadSubtitle, function (err, tracks) {
+  parallel(files, loadSubtitle, function (err, tracks) {
```

Comment:
Does this actually work? `parallel` works a bit differently than `async.map`. It takes an array of functions to execute.

I think this should be:

```js
var tasks = files.map(function (file) {
  return function (cb) {
    loadSubtitle(file, cb)
  }
})
parallel(tasks, function (err, tracks) {
  // ...
})
```

Or with arrow functions:

```js
parallel(files.map((file) => (cb) => loadSubtitle(file, cb)), function (err, tracks) {
  // ...
})
```